src/components/Select/Select.js

```diff
@@ -49,6 +99,12 @@ Select.propTypes = {
   title: PropTypes.string,
   className: PropTypes.string,
   isCreatable: PropTypes.bool,
+  formChangeHandler: PropTypes.func,
+  currentError: PropTypes.string,
+  formTouched: PropTypes.bool,
+  labelCopy: PropTypes.string,
```

Comment:
Should we make `labelCopy` required like many of the other components do? I think this could also make the related conditional rendering simplified as well in that you wouldn’t have to check it is defined

Reply:
The reason why I didn't make it required is because the Select component has the title prop which is essentially a label that lives within the border of the select box. So you wouldn't want both the title and labelCopy, but I thought if you weren't using the select box in a form, you may want the title prop instead of labelCopy. I wasn't planning on adding labelCopy until I realized that it looks inconsistent with the other fields. But, it does make the component more confusing to use. What do you think?

Reply:
@roblevintennis I would agree with this. I think in the majority of form elements we have implemented in the `EDS`, the label is required and set in the component.

Reply:
Sounds good! I'm gonna go ahead and make labelCopy and name required.

Reply:
Scratch that ^ Leaving it flexible for now after discussion.